homepage/templates/homepage/reviews/reviews.html

```diff
@@ -0,0 +1,18 @@
+<html>
+<head>
+<title>Class Rater Reviews</title>
+</head>
+
+<body>
+{% extends 'homepage/app_layout.html' %}
+{% block content %}
+<div class="container" style="margin-top:30px">
+  <div class="row">
+    <div class="col-sm-4">
+      <h2>Reviews Page</h2>
+    </div>
+  </div>
+</div>
+{% endblock %}
+<p class="EOM">(End of messages)</p>

```

Comment:
same as above, though here it could be relevant - so just move it into the block content?

Reply:
Moved into the block content